src/Futhark/Pass/GALOP/Layout.hs

```diff
@@ -1,4 +1,4 @@
-module Futhark.Pass.OptimiseArrayLayout.Layout (layoutTableFromIndexTable, Layout, Permutation, commonPermutationEliminators, LayoutTable) where
+module Futhark.Pass.GALOP.Layout (layoutTableFromIndexTable, Layout, Permutation, commonPermutationEliminators, LayoutTable) where
```

Comment:
Just FYI, I am going to revert this when we merge your PR.  It would be a mess if every pass had a cute name.